src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
It's better to avoid wildcard imports to prevent any unexpected behavior and conflicts. Import only the classes that are required.